docs/modules/ROOT/pages/push.adoc

```diff
@@ -0,0 +1,94 @@
+== AeroGear JavaScript Unifiedpush SDK
+
+An SDK that allows you to receive push notifications and integrate with the Aerogear Unifiedpush Server. In contrast to the other SDKs, `@aerogear/push` itself does not implement the handling of push notifications. It helps you to integrate a third party push library with our backend, the Aerogear Unifiedpush Server. While you can use any push plugin you like that supports Firebase Cloud Messaging and Apple Push Notification Services we suggest link:https://github.com/phonegap/phonegap-plugin-push[Phonegap Plugin Push] for Cordova and Ionic Apps.
```

Comment:
`In contrast to the other SDKs, `@aerogear/push` itself does not implement the handling of push notifications.` - Other SDK's do not do that either.

Reply:
`While you can use any push plugin you like that supports Firebase Cloud Messaging and Apple Push Notification Services we suggest link:https://github.com/phonegap/phonegap-plugin-push[Phonegap Plugin Push] for Cordova and Ionic Apps.`

Reply:
@wtrocki Do you think i should remove that sentence? Should we even mention the possibility of using other push plugins?

Reply:
Plus let's add. Please refer to plugin documentation for more info.

Reply:
> Should we even mention the possibility of using other push plugins?

No. We always tend to mention what we test and support only.